src/GameLogic/DefaultDropGenerator.cs

```diff
@@ -63,7 +63,7 @@ public IEnumerable<Item> GenerateItemDrops(MonsterDefinition monster, int gained
                     map.DropItemGroups,
                     questGroups)
                 .Where(group => IsGroupRelevant(monster, group))
-                .OrderBy(group => group.Chance);
+                .OrderByDescending(group => group.Chance);
```

Comment:
I understand your change, but I have the feeling that it will cause, that groups with a low chance will never drop, once the total sum gets over 1.
However, it's difficult to handle the new logic in an optimal way with the previous OrderBy, too. 👀
I need some time to think about that 😅